src/common/translation/i18n/fi.json

```diff
@@ -559,7 +559,7 @@
       "child": {
         "birthdate": {
```

Comment:
birthdate -> birthyear

Reply:
fixed. but still i would want to go through translations